website/www/site/content/en/documentation/sdks/python-pipeline-dependencies.md

```diff
@@ -146,8 +146,12 @@ Dataflow, see [Pre-building the python SDK custom container image with extra dep
 
 Pickling in the Python SDK is set up to pickle the state of the global namespace. By default, global imports, functions, and variables defined in the main session are not saved during the serialization of a Beam job.
 Thus, one might encounter an unexpected `NameError` when running a `DoFn` on any remote runner using portability. To resolve this, manage the main session by
-simply setting the main session. This will load the pickled state of the global namespace onto the Dataflow workers.
+simply setting the main session, `--save_main_session`. This will load the pickled state of the global namespace onto the Dataflow workers.
```

Comment:
```suggestion
setting the `--save_main_session` pipeline option. This will load the pickled state of the global namespace onto the Dataflow workers.
```